src/services/game-logic/pieces/Knight.ts

```diff
@@ -0,0 +1,23 @@
+import { PieceNames } from '../../../enums';
+import { Square } from '../../../models/Square';
+import { Piece } from './Piece';
+
+class Knight extends Piece {
+    name = PieceNames.KNIGHT;
+
+    getPossibleMoves(): Square[] {
+        let possibleRowMoves: number[] = [2, 1, -1, -2, -2, -1, 1, 2];
+        let possibleColumnMoves: number[] = [1, 2, 2, 1, -1, -2, -2, -1];
```

Comment:
całość wygląda super. z uwag, to nasuwa mi się tylko to, co sama wyniosłam z code review, czyli żeby skorzystać z inferencji typów i nie wypisywać ich tam, gdzie TS się domyśli

Reply:
dzięki ^^ poprawiłam